src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -512,6 +514,8 @@ protected void addTranslations()
 		addTooltip(MSItems.WOODEN_SPOON, "Like the kind you would eat your soup with");
 		add(MSItems.SILVER_SPOON, "Silver Spoon");
 		addTooltip(MSItems.SILVER_SPOON, "Could be the first silver object you've seen so far");
+		add(MSItems.NOSFERATU_SPOON, "Nosferatu Spoon");
+		addTooltip(MSItems.NOSFERATU_SPOON, "The life draining bloodlust of evil, in spoon form");
```

Comment:
life-draining?

Reply:
Just to be dramatic. It also serves to help compare it to the Nosferatu spell in Fire Emblem games. Regardless it is not essential

Reply:
I meant it as a suggestion to write life-draining instead of life draining